README.md

```diff
@@ -1,18 +1,29 @@
 # cluster-api-upgrade-tool
 
-## WARNING
+## Overview
 
-This tool is a work in progress. It may have bugs. **DO NOT** use it on a production Kubernetes cluster or any cluster you can't live without.
+This is a standalone tool to orchestrate upgrading the control plane machines of Kubernetes clusters created by Cluster
+API v0.2.x / API version v1alpha2.
 
-To date, we have only tested this with [Cluster API Provider for AWS](http://github.com/kubernetes-sigs/cluster-api-provider-aws) (CAPA)
+Our goal is to ultimately add this upgrade logic to Cluster API itself, but given that v1alpha2 doesn't easily lend
+itself to handling upgrades of control plane machines, we decided to build a temporarily tool that can fill that gap.
+Once Cluster API supports full lifecycle management of control planes, we plan to sunset this tool.
 
-## Overview
+We have tested this with [Cluster API Provider for AWS](http://github.com/kubernetes-sigs/cluster-api-provider-aws)
+(CAPA), and [Cluster API Provider for Docker](http://github.com/kubernetes-sigs/cluster-api-provider-docker) (CAPD).
+
+## Warning!
 
-This is a standalone tool to orchestrate upgrading Kubernetes clusters created by Cluster API v0.2.x / API version v1alpha2.
+While we've made every effort to build a robust tool that can perform upgrades correctly, including resuming a
+partially-completed or interrupted upgrade, there is a chance that things might go awry. Therefore:
 
-Our goal is to ultimately add this upgrade logic to Cluster API itself, but given that v1alpha2 doesn't easily lend itself to
-handling upgrades of control plane machines, we decided to build a temporarily tool that can fill that gap. Once Cluster API
-supports full lifecycle management of control planes, we plan to sunset this tool.
+**BACK UP** your control plane before you attempt an upgrade. This may include:
+
+- `Cluster` resource
+- `Machine` resources
+- `KubeadmConfig` resources
+- `Secret` resources containing the certificates and kubeconfig for your cluster
+- Contents of the `kube-system` namespace in the workload cluster (and any other namespaces/resources you may want)
```

Comment:
This should likely also call out either taking an etcd snapshot or doing a full archive using Velero of the workload cluster being upgraded.

Reply:
Will do. Thanks!

Reply:
Added a to-squash commit - ptal!

Reply:
the changes lgtm

Reply:
Squashed